CoffeeChat/Controller/EditDemographicsViewController.swift

```diff
@@ -11,18 +11,20 @@ import UIKit
 class EditDemographicsViewController: UIViewController {
 
     // MARK: - Private Data Vars
-    // TODO: Retrieve values from UserDefaults after saving from backend
-    private let name = "Lucy Xu"
-    private let year = "2021"
-    private let major = "Computer Science"
-    private let hometown = "Boston, MA"
-    private let pronouns = "She/Her/Hers"
-    // TODO: Update with networking values from backend
     private var classSearchFields: [String] = []
     private var fieldsEntered: [Bool] = [true, true, true, true, true] // Keep track of fields that have been entered
```

Comment:
https://www.tutorialspoint.com/swift/swift_arrays.htm

try something like `private var fieldsEntered: [Bool] = [Bool](count: 5, repeatedValue: true)`?

but this is also weird, can you explain how this is working? Can't you just check the fieldMap or something, i feel this is redundant in some way